dashboard/db/migrate/20160225192540_add_serialized_attributes_and_type_to_tasks.rb

```diff
@@ -0,0 +1,8 @@
+class AddSerializedAttributesAndTypeToTasks < ActiveRecord::Migration
+  def change
+    add_column :plc_tasks, :type, :string, null: false, default: 'Plc::Task'
+    add_column :plc_tasks, :properties, :text
+
+    Plc::Task.update_all(properties: '{}')
```

Comment:
hm, it's weird that you have to do this... I would assume that the serialized properties stuff would handle that (see https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/db/migrate/20150312000253_add_properties_to_user.rb and https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/db/migrate/20150521231522_add_properties_to_scripts.rb) did it break when you didn't?